lib/tasks/update_study_type_question_text.rake

```diff
@@ -1,10 +1,10 @@
 namespace :update_study_type_question_text do
   desc "TODO"
   task change_question_task: :environment do
-    @stq = StudyTypeQuestion.where(question: "1. Does your Informed Consent provide information to the participant specifically stating their study participation will be kept private from anyone outside the research team? (i.e. your study has a Certificate of Confidentiality or involves sensitive data collection which requires de-identification of the research participant in Epic.)")
+    @stq = StudyTypeQuestion.where(friendly_id: "certificate_of_conf").second
```

Comment:
Confused as to how this works.  It finds by friendly_id and then grabs the second record it finds.  Is that correct?